src/Node.js

```diff
@@ -0,0 +1,26 @@
+class Node {
```

Comment:
utilize OOP. let `Node` be a superclass and `Event`, `Action`, `CustomerUpdate` and `Condition` subclasses. `Node` will have constructor `constructor(id, ...params)` and then we can have `class Event extends Node ...`